server/routes/api/settings.ts

```diff
@@ -263,6 +263,23 @@ settingsRoutes.route("/email_content/:type")
 		}
 	});
 
+settingsRoutes.route("/email_content/batch_email/rendered")
+	.post(isAdmin, uploadHandler.any(), async (request, response) => {
+		// Endpoint for generalized email, without templating for user args; meant for batch sending email
+		try {
+			let markdown: string = request.body.content;
+			let html: string = await renderEmailHTML(markdown, request.user as IUser);
+			let text: string = await renderEmailText(html, request.user as IUser, true);
+			response.json({ html, text });
+		}
+		catch (err) {
+			console.error(err);
+			response.status(500).json({
+				"error": "An error occurred while rendering the email content"
+			});
+		}
+	});
+
 settingsRoutes.route("/email_content/:type/rendered")
```

Comment:
This is literally the same function code. Could we simplify this?

Reply:
Yep, looks like you don't actually use :type for anything so it works with the method removed.

Reply:
Meant more that we might want to make this a function that we pass to those two event handlers instead of repeating twice

Reply:
Nevermind you did fix this